proto/store/approval_template.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+option go_package = "generated-go/store";
+
+message ApprovalFlow {
+  repeated ApprovalStep steps = 1;
```

Comment:
Might need an field to indicate the type of approval flow
<img width="574" alt="image" src="https://user-images.githubusercontent.com/2749742/226523266-e1fba689-92be-496f-b1a6-3ea7f083f18c.png">